### DISCUSSION TITLE
[Feature] it will be better to show the area size or pixel coordinate

### DISCUSSION THREAD

**Discussion by junxnone** [wkentaro/labelme#1123] [Ideas / Feature requests]
> Thanks for your share.

**Is your feature request related to a problem? Please describe.**
When I label or check the small object , I want to see if this is the limit of my algo

**Describe the solution you'd like**
- show the area size/width/height after the bbox or polygon...
- show the pixel coordinate of the image dynamically with mouse pointer

**Comment by ManuelZ** [comment]
I was just looking for this feature for the exact same reason as @junxnone

**Comment by ManuelZ** [comment]
This can be done for rectangles adding this:
```
if len(selected_shapes) == 1:
    shape = selected_shapes[0]
    if shape.shape_type == "rectangle" and len(shape.points) == 2:
        unscaled_rectangle = QtCore.QRectF(shape.points[0], shape.points[1])
        w = abs(unscaled_rectangle.width()) + 1
        h = abs(unscaled_rectangle.height()) + 1
        self.status(f"Shape h={h:.0f}px; w={w:.0f}px; area={w*h:.0f} px^2")
```

Just below this line:
https://github.com/wkentaro/labelme/blob/0ea3e8e05a247b87eacdd0bc13a1c737509bf193/labelme/app.py#L1298

To use it, select a rectangle (first click on "Edit Polygons" to be able to select polygons) and the message will appear in the status bar. It's not perfect in terms of usability, but is a starting point.
